### PR TITLE
Features

### PR DESCRIPTION
**Pull request by Fanger53**
# Responsive design of the next web page
This project consists of building a clone of the next web page, to understand the use of media-queries and how works the responsive design in differents electronic dispositive

### CODE REVIEW

**Review by MuriloRoque — COMMENTED** [review]
# Status: Invalid :red_circle: 

### Hello @Fanger53 and @ositaadimma :wave: 

### I am marking this project as invalid for the following reasons:

- [ ] Your Pull Request needs to have a meaningful title and message, please edit it so it becomes more professional. You should describe what is the purpose of the project and what you did to complete it :+1: 

- [ ] Your README file should be more professional too, please add a screenshot and format it. You can use [this](https://github.com/microverseinc/readme-template) template if you like :smile: 

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by Fanger53** [comment]
the readme file was updated  with the screenshots of the project

File: README.md [hunk 1]
```diff
@@ -1 +1,51 @@
...
+## movile
+
+<img width="452" alt="mobile-img" src="https://user-images.githubusercontent.com/31552010/92154426-64a73a80-edeb-11ea-81b8-b3f0c94d3780.png">
+
+
+
+
+Built With : Html,CSS and github technologies used:flex-box and grid of CSS, Using StyleLinters.
+
+Live Demo Link:https://raw.githack.com/Fanger53/responsive-design/tree/features
```
Comment (SpaYco) [inline README.md]: - [ ] never add something without testing, your demo link doesn't work, you need to use the link for `index.html` not the repo, in order for it to work.

File: README.md [hunk 2]
```diff
@@ -1 +1,51 @@
...
+
+Authors
+
+👤 Author1
+
+Github: https://github.com/ositaadimma
+
+Twitter:
+
+Linkedin:
```
Comment (SpaYco) [inline README.md]: - [ ] if you don't have links, remove this.

File: css/media-querys.css [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  .section-four-grid {
+    grid-template-columns: 50% 50%;
+    grid-template-rows: repeat(4, 730px);
+  }
+}
+
+/* Desktop */
+
+@media (min-width: 1040px) {
+  /* nav bar size desktop */
```
Comment (SpaYco) [inline css/media-querys.css]: - [ ] desktop size is 1024px

File: css/media-querys.css [hunk 4]
```diff
@@ -0,0 +1,200 @@
+/* Tablet */
+
+@media (min-width: 790px) {
+  /* nav-bar */
```
Comment (SpaYco) [inline css/media-querys.css]: - [ ] this should be 768px

File: index.html [hunk 5]
```diff
@@ -0,0 +1,898 @@
...
+        <section class="section-three  center ">
+            <div class="section-three-title-flex center list ">
+              <div class="titles ">
+                <a href="#">Latest funding rounds on index.co <i class="fas fa-angle-right"></i></a>
+              </div>
+              <div class="right-title">
+                <a href="#" class="list-b">Add info</a>
+                <a href="#" class="list-b">Get update by email</a>
+                <a href="#" class="list-b last-a" >See more</a>
+              </div>
```
Comment (SpaYco) [inline index.html]: on large screens: 

- [ ] the small titles should be on the right in one line

| Yours | Original |
| --- | --- |
| image | image |

on medium screens: 

- [ ] the small titles should on the same line under the big title, with borders in between

| Yours | Original |
| --- | --- |
| image | image |

File: index.html [hunk 6]
```diff
@@ -0,0 +1,898 @@
...
+                        </span>
+                      </a>
+                    </div>
+                    <div  class="section-three-header-sections ">
+                      <a href="#" class="s-3-h-s-a " > Asia Dealbook: Potato Play raises $1.75m from BEENEXT; PriceOye.pk</a>
+                      
+                      <a href="#" class="s-3-h-s-a " >raises pre-Series A <span class="color-gray"> on DealStreetAsia</span></a>
+                    </div>
+              </div>
+            </div>
```
Comment (SpaYco) [inline index.html]: - [ ] change the colors of the links

| Yours | Original |
| --- | --- |
| image | image |

File: index.html [hunk 7]
```diff
@@ -0,0 +1,898 @@
...
+            <li class="list-li"><a href="#" class="footer-list-a" >EVENTS </a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ABOUT</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ADVERTISE</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">JOBS</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">CONTACT</a></li>
+          </ul>
+        </div>
+        <div class="footer-legal" >
+            <div class=" legal-text center"><span ><img src="./image/tnw.svg" alt="logo" class="footer-legal-logo"></span><p class="footer-legal-text">© 2006–2020 The Next Web B.V.</p> <p class="footer-legal-text">Made with <span><i class="fas fa-heart"></i></span> in Amsterdam.</p><p><p class="footer-legal-text">Powered by</p> <span><img src="./image/stackpath.svg" alt="logo" class="footer-legal-logo2"></span></div>
+        </div>
```
Comment (SpaYco) [inline index.html]: - [ ] these should be in one line on both large and medium screens, and 3 lines on small screens, yours changes on the medium screen

File: index.html [hunk 8]
```diff
@@ -0,0 +1,898 @@
...
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fab fa-pinterest-p "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-wifi "></i></a></li>
+            <li class="list-li"><a href="#" class="footer-social icon"><i class="fas fa-envelope "></i></a></li>
+          </ul>
+          <ul class="list footer-list center">
+            <li class="list-li"><a href="#" class="footer-list-a" >EVENTS </a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ABOUT</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">ADVERTISE</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">JOBS</a></li>
+            <li class="list-li"><a href="#" class="footer-list-a">CONTACT</a></li>
```
Comment (SpaYco) [inline index.html]: - [ ] the contact icon goes to the next line on small screens
image

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

* [ ] Pull Request should have  a meaningful title ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

- [ ] Files not related to the project should not be committed and added to .gitignore (e.g. `package-lock.json`) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

> I Added more required changes below

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Changes requested :red_circle: 

### Hello again @Fanger53 and @ositaadimma :wave: 

### Nice job implementing most of the changes requested by the previous TSE, but we need to fix one thing that was left behind after his review:

- [ ] **(MANDATORY)** Your footer links are still going above each other, check the screenshots below for medium and small screens:

Screenshot from 2020-09-03 17-59-39
<br>
Screenshot from 2020-09-03 17-59-36
<br>

### The next suggestions are marked as **OPTIONAL**, since they weren't requested by the previous TSE:

- [ ] **(OPTIONAL)** On large screens, your page is not centered, it's not covering the entire width, like in the original:

Screenshot from 2020-09-03 17-58-35
<br>

- [ ] **(OPTIONAL)** There should be a TNW logo inside of this big image at the top of the page, below the navbar, please check the original:

Screenshot from 2020-09-03 17-59-01
<br>

- [ ] **(OPTIONAL)** On very small screens, the content of your page doesn't fit the outer container, since this container gets too small, see below:

Screenshot from 2020-09-03 17-59-46
<br>

- [ ] **(OPTIONAL)** You are using a lot of repeated properties in your CSS (check below), please make classes for these properties, so your file can be more professional and organized :smiley: 

 12 |   color: white;
      10 |   font-weight: 700;
       9 |   width: 100%;
       8 |   align-items: center;
       7 |   width: 95%;
       6 |   color: black;
       6 |   display: grid;
       5 |   color: red;
       5 |   display: inline-block;
       5 |   display: none;
       5 |   justify-content: space-between;

### Kindly make the requested changes and submit for another review, please :pray: 

### I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make a few more changes!

## Required Changes ♻️ 

- [ ] on small screens the content of the page is taking more than the screen, make it always take 100% of the screen 
image

- [ ] the search button on the navbar should be on the very right at the medium screen, on small screens,
 it should be with a hamburger.
 they exist but they are not on the right, if I widen the screen it will have a space on the side, and if I shrink it, the buttons will go inside the screen

image
image

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

### Hello again @Fanger53 and @ositaadimma :wave: 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**